doc/source/ray-air/api/serve.rst

```diff
@@ -0,0 +1,18 @@
+.. _air-serve-integration:
+
+Model Serving in AIR
```

Comment:
Add `ray.serve.air_integrations.PredictorWrapper`?

Reply:
PredictorWrapper should be on this page already.

Reply:
Ahhh woops the rendering looked a bit strange because of the alias

<img width="827" alt="image" src="https://user-images.githubusercontent.com/3967392/218189697-3a37a701-8170-441e-8f44-cf13400a848b.png">

Reply:
My mistake!

Reply:
Yeah, to be fair this page is kinda a mess. The see also tooltip + autosummary + autofunction is a bit much. But I had to do switch predictor deployment to an autofunction because it's an empty class that was erroring if I put it in autosummary.